gum/backend-arm64/gumstalker-arm64.c

```diff
@@ -3109,6 +3118,7 @@ gum_stalker_iterator_put_callout (GumStalkerIterator * self,
                                   GDestroyNotify data_destroy)
 {
   GumExecBlock * block = self->exec_block;
+  GumExecBlock * active_block = gum_exec_block_get_active_block(block);
```

Comment:
This also seems like a symptom and not the correct fix. It seems the code generation is happening on the wrong block. Maybe it's just another symptom of failing to update the mappings?